samples_project/Assets/SampleViewer/Samples/Measure/readme.md

```diff
@@ -2,13 +2,19 @@
 
 Draw lines on the map and show distance.
 
-![Measure](Measure.PNG)
+![Measure](measure.PNG)
+
+## How to use the sample
+
+1. Set your API Key in the ArcGISMapActor using the World Outliner window (if you are using the SampleViewerLevel you can also set the key through the UI or the level blueprint).
+2. While holding shift, click on the map to begin measuring. Continue to click on the map to add additional vertices. The accumulated distance will be shown. This sample is only set up to work with a mouse and keyboard.
```

Comment:
```suggestion
1. Open the **Measure** scene.
2. Set your API Key in the **ArcGIS Map** game object using the **Inspector** window.
3. While holding shift, click on the map to begin measuring. Continue to click on the map to add additional vertices. The accumulated distance will be shown. This sample is only set up to work with a mouse and keyboard.
```

I didn't doublecheck with the Unity screen though I guess it's something like this.